From: Mark Kretschmann <kretschmann@kde.org>
Date: Sun Aug 18 12:22:35 2013 +0200
Bug: http://bugs.kde.org/323632
Origin: backport, commit e631534993c03244a36ed013c186ad8bf83ca1fd
Applied-Upstream: 2.8.1

    Don't add the analyzer applet when Phonon doesn't support it.
    
    Stupid... I forgot about that file.
    
    BUG: 323632
    BACKPORT

--- a/data/amarok_homerc
+++ b/data/amarok_homerc
@@ -1,3 +1,3 @@
 [Containment 0]
 firstShowingApplet=0
-plugins=currenttrack,analyzer,wikipedia,lyrics
+plugins=currenttrack,wikipedia,lyrics
